core/tests/loadnext/src/sdk/ethereum/mod.rs

```diff
@@ -439,6 +479,7 @@ impl<S: EthereumSigner> EthereumProvider<S> {
 
     /// Performs a deposit in zkSync network.
     /// For ERC20 tokens, a deposit must be approved beforehand via the `EthereumProvider::approve_erc20_token_deposits` method.
+    /// Cannot deposit ETH onto custom base token chains.
```

Comment:
We can deposit Eth to custom base token chains though. I am not sure what you meant here?

Reply:
I meant this method can't.